public/app/features/explore/Logs/LogsTableWrap.tsx

```diff
@@ -308,6 +363,74 @@ export function LogsTableWrap(props: Props) {
 
     // Update url state
     updatePanelState(newPanelState);
+  }
+
+  // Toggle a column on or off when the user interacts with an element in the multi-select sidebar
+  const toggleColumn = (columnName: fieldName) => {
+    if (!columnsWithMeta || !(columnName in columnsWithMeta)) {
+      console.warn('failed to get column', columnsWithMeta);
+      return;
+    }
+
+    const length = Object.keys(columnsWithMeta).filter((c) => columnsWithMeta[c].active).length;
+
+    const isActive = !columnsWithMeta[columnName]?.active ? true : undefined;
```

Comment:
```suggestion
    const isActive = !columnsWithMeta[columnName].active ? true : undefined;
```
Shouldn't be needed

Reply:
Yeah I was looking at the wrong code 🤦  https://github.com/grafana/grafana/pull/79536/commits/d8866d276d08b80aa823cd5515f48c0297197790

Sorry about that. Fixed and pushed!